### PR TITLE
Updated SimpleCov configuration

### PR DESCRIPTION
**Pull request by jhsu802701**
I made the following changes to the SimpleCov configuration:
* Filtered out the db and spec directories
* Provided grouped coverage results

### CODE REVIEW

**Comment by compwron** [comment]
@jhsu802701 What change does this make to the output of simplecov? Please add before/after screenshots or similar.

While you're in there, if you could add the simplecov publisher to our github actions that would be very useful! (It got removed when we switched from travisci to github actions.) https://github.com/rubyforgood/casa/pull/1288 https://github.com/rubyforgood/casa/issues/1304

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,7 +1,19 @@
 require "simplecov"
 require "pry"
 SimpleCov.start do
-  track_files "{app,lib}/**/*.rb"
+  add_filter '/db/'
```
Comment (compwron) [inline spec/spec_helper.rb:4]: We do actually want to assert coverage on `db/seeds` so please don't filter out all of `db`

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
+  add_group "Controllers", "app/controllers"
+  add_group "Channels", "app/channels"
+  add_group "Decorators", "app/decorators"
+  add_group "Helpers", "app/helpers"
+  add_group "Jobs", "app/jobs"
+  add_group "Importers", "app/lib/importers"
+  add_group "Mailers", "app/mailers"
+  add_group "Policies", "app/policies"
+  add_group "Values", "app/values"
+  add_group "Libraries", "lib"
```
Comment (compwron) [inline spec/spec_helper.rb:16]: If this grouping changed the output in a nice way, let's also add a group for `after_party` tasks, which should have coverage.

Comment (jhsu802701) [reply spec/spec_helper.rb]: I replaced the "lib" group with a "/lib/tasks" group.

File: spec/spec_helper.rb (hunk 1 above)
Comment (jhsu802701) [reply spec/spec_helper.rb]: I removed the "/db" filter.

**Comment by compwron** [comment]
I ran it locally rather than requesting again @jhsu802701 to post a screenshot, it doesn't break anything. I'll merge it. 

@jhsu802701 if you could add the simplecov publisher to our github actions that would be very useful! (It got removed when we switched from travisci to github actions.) 
PR: #1288 
github issue: #1304

**Review by compwron — APPROVED** [review]

**Pull request merged**